packages/clay-table/stories/index.tsx

```diff
@@ -31,41 +31,23 @@ function ClayCheckboxWithState(props: any) {
 	);
 }
 
-const DropDownWithState: React.FunctionComponent<any> = ({
-	children,
-	...others
-}) => {
-	const [active, setActive] = React.useState(false);
-	return (
-		<ClayDropDown
-			{...others}
-			active={active}
-			alignmentPosition={Align.BottomRight}
-			onActiveChange={newVal => setActive(newVal)}
-			trigger={<ClayIcon spritemap={spritemap} symbol="ellipsis-v" />}
-		>
-			{children}
-		</ClayDropDown>
-	);
-};
-
 const thumbnail = require('./static/thumbnail_coffee.jpg');
 
-function ClayDropdownWithTrigger() {
+function Dropdown() {
 	return (
-		<DropDownWithState>
-			<ClayDropDown.ItemList>
-				{[
-					{href: '#1', label: 'One'},
-					{href: '#2', label: 'Two'},
-					{href: '#3', label: 'Three'},
-				].map((item, i) => (
-					<ClayDropDown.Item href={item.href} key={i}>
-						{item.label}
-					</ClayDropDown.Item>
-				))}
-			</ClayDropDown.ItemList>
-		</DropDownWithState>
+		<ClayDropDownWithItems
+			items={[
+				{href: '#1', label: 'One'},
+				{href: '#2', label: 'Two'},
+				{href: '#3', label: 'Three'},
+			]}
+			spritemap={spritemap}
+			trigger={
+				<ClayButton className="component-action" monospaced>
```

Comment:
Add `displayType="unstyled"`, the button flashes with the blue background in the hover.

Reply:
How did you catch it? Ninja

Reply:
Actually not the hover is when you click the button. 🙂